account/accounts.go

```diff
@@ -272,15 +212,24 @@ func (m *Manager) SetChatAccount(privKey *ecdsa.PrivateKey) {
 	}
 }
 
-// SelectedWalletAccount returns currently selected wallet account
-func (m *Manager) SelectedWalletAccount() (*SelectedExtKey, error) {
+// MainAccountAddress returns currently selected watch addresses.
+func (m *Manager) MainAccountAddress() (common.Address, error) {
```

Comment:
is this address always the a single account or it can be changed based on users preferences?

Reply:
@dshulyak  it's passed from status-react to the Login function, so I think it will be configurable from there

Reply:
so we cannot use it as a unique account identifier?

Reply:
yes we can use any id to identify the "user account", but this is just an ethereum address used as main account address for the user

Reply:
@dshulyak it's only used as account to be returned from web3.eth.getAccounts